terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -543,17 +542,14 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 "cidr_block": f"${{cidrsubnet(aws_vpc.gitlab.cidr_block, 8, {subnet_number(zone, public)})}}",
                 "map_public_ip_on_launch": public,
                 "vpc_id": "${aws_vpc.gitlab.id}",
-                "tags": {
-                    "Name": f"azul-gitlab-{subnet_name(public)}-{subnet_number(zone, public)}"
-                }
+                "tags": tags(f"azul-gitlab-{subnet_name(public)}-{subnet_number(zone, public)}",
+                             component="azul-gitlab")
             } for public in (False, True) for zone in range(num_zones)
         },
         "aws_internet_gateway": {
             "gitlab": {
                 "vpc_id": "${aws_vpc.gitlab.id}",
-                "tags": {
-                    "Name": "azul-gitlab"
-                }
+                "tags": tags("azul-gitlab", component="azul-gitlab")
```

Comment:
It seems like this is screaming for a `gitlab_tags()` wrapper around `tags()` so the component tag doesn't need to be duplicated all over the place.